scripts/xtensa-build-all.sh

```diff
@@ -57,11 +62,15 @@ while getopts "rudj:ck:vo:a" OPTION; do
 		k) PRIVATE_KEY=$OPTARG ;;
 		v) BUILD_VERBOSE='VERBOSE=1' ;;
 		a) PLATFORMS=("${SUPPORTED_PLATFORMS[@]}") ;;
+		m) MEU_TOOL_PATH=$OPTARG ;;
+		s) MEU_OPEN_SSL_PATH=$OPTARG ;;
 		*) print_usage; exit 1 ;;
         esac
 done
 shift $((OPTIND-1))
 
+SIGNING_TOOL=$(echo $SIGNING_TOOL | tr '[:upper:]' '[:lower:]')
+
```

Comment:
I don't understand when this case conversion is needed or why, can you explain? Also don't understand why the default RIMAGE value is far away at the very top.